cmd/server-main.go

```diff
@@ -260,6 +260,8 @@ func isDistributedSetup(eps []*url.URL) bool {
 	return false
 }
 
+var windowsDriveMatch = regexp.MustCompile("^[a-zA-Z]$")
```

Comment:
Add a comment here, also i thought we are going to explore a non-regex approach here..